flax/linen/normalization.py

```diff
@@ -730,6 +730,122 @@ def __call__(self, x, mask=None):
     )
 
 
+class InstanceNorm(Module):
```

Comment:
Keras implements [InstanceNormalization](https://www.tensorflow.org/addons/api_docs/python/tfa/layers/InstanceNormalization) as a special case of `GroupNorm`. Kinda makes sense, wondering if we could do the same?

Reply:
I will look into this